assets/js/buttons-examples.js

```diff
@@ -807,6 +807,13 @@ $(document).ready(function () {
           totalWrittenClasses += ` .${thisButtonClasses}`;
```

Comment:
If you look at this line @Hanaffi you will see that the problem is here. We're just adding a `.` to `thisButtonClasses`. However, `thisButtonClasses` can have many classes in it. For example `spin left purple-btn`. 

In the example given, `totalWrittenClasses` expected value should be `.spin .left .purple-btn` but in our code it will be `.spin left purple-btn` which means only spin has a `.` before.

What I'm saying this line needs to be changed. In `thisButtonClasses` spaces should be replaced with ` .` which is a space followed by a `.` and a `.` need to be added in the beginning of it. This way `spin left purple-btn` will become `.spin .left .purple-btn`.